src/inferences/junctionTree/propagate-potentials.ts

```diff
@@ -47,10 +42,11 @@ interface IDistributeEvidenceOrder {
   neighborId: string;
 }
 
-const hasSepSetCliques = curry((cliqueIdA: string, cliqueIdB: string, sepSet: ISepSet) =>
+const hasSepSetCliques: (cliqueIdA: string, cliqueIdB: string) => (sepSet: ISepSet) => boolean =
```

Comment:
added an explicit type signature.   Some of the tests were failing to compile because they could not infer the correct type.